test/integration/crud/bulk.test.js

```diff
@@ -7,7 +7,12 @@ const {
   ignoreNsNotFound,
```

Comment:
I think we also need a regression test here.  We've added two unit tests covering the behavior we modified but we should add a test that proves that the original bug is no longer reproducible (this prevents a "regression" in the future, where we accidentally introduce the bug back into the driver).

This should be relatively straightforward.  We need to test that calling insertMany with a sparse array throws the useful error now.  Fortunately for us, Anna gave us a reproduction (the script included in the jira ticket) so we basically just need to turn the script into a test that asserts the correct error is thrown.

Reply:
ok so something similar to the tests i had written initially?

Reply:
I don't remember exactly how your tests looked initially, but if you deleted that work because of something I said I'm sorry 😬

Reply:
no its completely fine it wasn't anything major!